projects/components/src/datagrid/datagrid.component.spec.ts

```diff
@@ -478,123 +500,134 @@ describe('DatagridComponent', () => {
                 });
 
                 it('allows the height to be dynamically changed', function (this: HasFinderAndGrid): void {
-                    this.finder.hostComponent.height = 200;
+                    this.hostComponent.height = 200;
                     this.finder.detectChanges();
-                    expect(this.clrGridWidget.gridHeight).toEqual('200px');
-                    this.finder.hostComponent.height = undefined;
+                    expect(getGridHeight(this.clrGridWidget.unwrap().parent())).toEqual('200px');
+                    this.hostComponent.height = undefined;
                     this.finder.detectChanges();
-                    expect(this.clrGridWidget.gridContainerClasses).toContain('fill-parent');
-                    expect(this.clrGridWidget.gridHeight).toEqual('unset');
+                    expect(this.clrGridWidget.unwrap().parent().classes()).toContain('fill-parent');
+                    expect(getGridHeight(this.clrGridWidget.unwrap().parent())).toEqual('unset');
                 });
             });
         });
 
         describe('@Input() emptyGridPlaceholder', () => {
             it('does not show the placeholder while the grid is loading', function (this: HasFinderAndGrid): void {
                 this.finder.detectChanges();
-                expect(this.clrGridWidget.loading).toBeTruthy();
-                expect(this.clrGridWidget.placeholder).toEqual('');
+                expect(this.clrGridWidget.unwrap().elements[0].componentInstance.loading).toBeTruthy();
+                expect(this.clrGridWidget.getPlaceHolder().text()).toEqual('');
             });
 
             it('shows the placeholder if the grid is empty', function (this: HasFinderAndGrid): void {
                 this.finder.detectChanges();
-                expect(this.clrGridWidget.loading).toBeTruthy();
-                this.finder.hostComponent.gridData = {
+                expect(this.clrGridWidget.unwrap().elements[0].componentInstance.loading).toBeTruthy();
+                this.hostComponent.gridData = {
                     items: [],
                     totalItems: 0,
                 };
                 this.finder.detectChanges();
-                expect(this.clrGridWidget.loading).toBeFalsy();
-                expect(this.clrGridWidget.placeholder).toEqual('Placeholder');
+                expect(this.clrGridWidget.unwrap().elements[0].componentInstance.loading).toBeFalsy();
+                expect(this.clrGridWidget.getPlaceHolder().text()).toEqual('Placeholder');
             });
 
             it('does not show the placeholder if the grid has data', function (this: HasFinderAndGrid): void {
                 this.finder.detectChanges();
-                expect(this.clrGridWidget.loading).toBeTruthy();
-                this.finder.hostComponent.gridData = {
+                expect(this.clrGridWidget.unwrap().elements[0].componentInstance.loading).toBeTruthy();
```

Comment:
Is there anything else we can assert about the HTML to implement this instead of querying to the component?